cmake/fmtlib.cmake

```diff
@@ -0,0 +1,31 @@
+include(ExternalProject)
+
+if (${CMAKE_SYSTEM_NAME} STREQUAL "Emscripten")
+	set(FMTLIB_CMAKE_COMMAND emcmake cmake)
+else()
+	set(FMTLIB_CMAKE_COMMAND ${CMAKE_COMMAND})
+endif()
```

Comment:
`FMTLIB_CMAKE_COMMAND` seems unused. Is it really needed here?

Reply:
I don't think so, I think it was copied from ranges/jsoncpp where the ExternalProject actually uses it.